testsuite/tests/cxx-api/all_includes.ml

```diff
@@ -1,13 +1,41 @@
 (* TEST
- modules = "stubs.c";
- readonly_files = "all-includes.h";
- not-msvc;
- flags = "-ccopt -x -ccopt c++ -ccopt -std=c++11";
+ has-cxx;
+ not-clang;
+ hassysthreads;
+ readonly_files = "deps.sh all-includes.h.sh cxx-sarif.sh stubs.cpp";
+ include runtime_events;
+ include systhreads;
+ script = "sh ${test_source_directory}/deps.sh";
+ script;
+ {
```

Comment:
It's not instantly clear what the two steps are doing differently - a quick comment here that it's testing (I think) ocamlopt and then `-output-complete-exe`?